### PR TITLE
docs: Update CHANGELOG.md & Versioning

### PR DESCRIPTION
**Pull request by ColinKYuen**
### Summary

Update CHANGELOG.md & Versioning

### Description

Updating changelog to reflect changes:
* Added AWS IAM Database Authentication
* Bug fixes

### Additional Reviewers

@matthewh-BQ
@karenc-bq
@sergiyv-bitquill
@brunos-bq
@yanw-bq

### CODE REVIEW

**Review by sergiyv-bitquill — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+
+### Added
+  * Support for AWS IAM database authentication, please refer to the AWS IAM Database Authentication section in the README.md for more information
+
+### Fixes
+  * Using invalid login parameters after a valid set will no longer be able to establish a connection using cached information.
+  * Old Linux bug causing infinite loops in test fixed.
+
+### Misc
+  * Enforce source & target build using Java 8.
```
Comment (karenc-bq) [inline CHANGELOG.md:13]: This is a fix

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## Version 0.2.2 (Public Preview) - 2021-11-16
+
+### Added
+  * Support for AWS IAM database authentication, please refer to the AWS IAM Database Authentication section in the README.md for more information
```
Comment (karenc-bq) [inline CHANGELOG.md:6]: Full stop

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,19 +1,30 @@
...
+* Java 8 is now enforced in the build process.
+* Resolved an issue for when connecting with an invalid connection after a valid connection. Before this would allow the user to connect despite having invalid information, such as a username.
+* Bug fixes with flakey tests.
+
 ## [Version 0.2.0 (Public Preview)](https://github.com/awslabs/aws-mysql-jdbc/releases/tag/0.2.0) - 2021-08-30
 
 ### Added
-  * `setAwsProtocolOnly` changed to be static method.
-  * Merged upstream changes from MySQL 8.0.23 community driver.
+* `setAwsProtocolOnly` changed to be static method.
```
Comment (karenc-bq) [inline CHANGELOG.md:16]: I think this is more of an improvement.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,19 +1,30 @@
 # Changelog
 
+## [Version 0.3.0 (Public Preview)](https://github.com/awslabs/aws-mysql-jdbc/releases/tag/0.3.0) - 2021-11-18
+
+### Added
+* AWS IAM Authentication is now supported. Usage instructions can be found [here](https://github.com/awslabs/aws-mysql-jdbc#aws-iam-database-authentication).
+
+### Improvements
+* Java 8 is now enforced in the build process.
+* Resolved an issue for when connecting with an invalid connection after a valid connection. Before this would allow the user to connect despite having invalid information, such as a username.
```
Comment (karenc-bq) [inline CHANGELOG.md:10]: The second sentence looks a bit weird, what do you think of:
Users were able to connect with cached information after a valid connection despite providing invalid information.

**Pull request merged**